cScripts/functions/systems/fn_getArsenalWhitelist.sqf

```diff
@@ -37,26 +37,26 @@ private _companyItems = switch (_company) do {
 };
 
 
-private _roleSpecific = switch ([player] call EFUNC(gear,getLoadoutRole)) do {
-    case "officer": {GET_CONTAINER_KEYS("arsenal_role_officer");};
-    case "squadleader": {GET_CONTAINER_KEYS("arsenal_role_squadleader");};
-    case "fireteamleader": {GET_CONTAINER_KEYS("arsenal_role_fireteamleader");};
-    case "weapons": {GET_CONTAINER_KEYS("arsenal_role_weapons");};
-    case "pilot";
-    case "rotarypilot": {GET_CONTAINER_KEYS("arsenal_role_rotarypilot");};
-    case "rotarycrew": {GET_CONTAINER_KEYS("arsenal_role_pilotcrew");};
-    case "rotarycls": {
-        private _rotaryRole = GET_CONTAINER_KEYS("arsenal_role_pilotcrew");
-        private _clsRole = GET_CONTAINER_KEYS("arsenal_role_cls");
-        _rotaryRole + _clsRole;
+private _tagGear = [];
+{
+    private _tag = switch (_x) do {
+        case "role_officer": {GET_CONTAINER_KEYS("arsenal_role_officer");};
+        case "role_squadleader": {GET_CONTAINER_KEYS("arsenal_role_squadleader");};
+        case "role_fireteamleader": {GET_CONTAINER_KEYS("arsenal_role_fireteamleader");};
+        case "role_weapons": {GET_CONTAINER_KEYS("arsenal_role_weapons");};
+        case "role_pilot";
+        case "role_rotarypilot": {GET_CONTAINER_KEYS("arsenal_role_rotarypilot");};
+        case "role_rotarycrew": {GET_CONTAINER_KEYS("arsenal_role_pilotcrew");};
+        case "role_pilotfighter": {GET_CONTAINER_KEYS("arsenal_role_pilotfighter");};
+        case "role_pilottransport": {GET_CONTAINER_KEYS("arsenal_role_pilottransport");};
+        
+        case "tag_cls": {GET_CONTAINER_KEYS("arsenal_role_cls");};
+        case "tag_medic": {GET_CONTAINER_KEYS("arsenal_role_medic");};
+        case "tag_doctor": {GET_CONTAINER_KEYS("arsenal_role_doctor");};
```

Comment:
```suggestion
        case "tag_cls": {GET_CONTAINER_KEYS("arsenal_tag_cls");};
        case "tag_medic": {GET_CONTAINER_KEYS("arsenal_tag_medic");};
        case "tag_doctor": {GET_CONTAINER_KEYS("arsenal_tag_doctor");};
```

I would go into `init_logistics.sqf` and change these container names to match the suffixes we are looking for. in the switch here to maintain a consistent naming convention.